tools/travis/py3k

```diff
@@ -45,10 +45,18 @@ libpasteurize.fixes.fix_newstyle
 
 echo; echo "Testing for additions of Python 2 patterns we've removed as part of moving towards Python 3 compatibility."; echo
 
+script_dir=$(dirname $(realpath $BASH_SOURCE))
```

Comment:
Yeah you're probably right. I read it [here](stackoverflow.com/questions/24112727/shell-relative-paths-based-on-file-location-instead-of-current-working-director#comment-37199405) that "$0" might give some problems. And I thought that if the user runs py3k from the current directory, dirname might not behave properly in some cases.

But I think `$(dirname "$0")` is more readable. I'll change it.